docs/buckets.rst

```diff
@@ -0,0 +1,62 @@
+Buckets
+#######
+
+Buckets are a way to group users around some data.
```

Comment:
This definition doesn't say much. "Buckets are a group of collections, which can be shared accross users" seems more close to what I have in mind.